### PR TITLE
Milestone 1: Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by Braingenius2**
# Set up a "Hello Microverse" project
In this  milestone, we 

- Worked to understand the concept of version control.
- Followed GitHub flow.
- Opened pull requests in GitHub.
- Wrote descriptive, easy-to-understand commit messages.
- Wrote short, detailed, easy-to-understand descriptions of pull requests.
- Wrote a descriptive and easy-to-understand README in business English for every project submitted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,259 @@
...
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (huseyinbiyik) [inline README.md]: Please add a software license to your project. In this way, you can present each of your repositories professionally to the visitors. You can check the link to learn how to add a license (bottom of the page: https://github.com/microverseinc/readme-template/blob/master/README.md#-license-

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
- Please set up linters as requested on project requirements. You can follow the instruction to set them up on this [page](https://github.com/microverseinc/linters-config). There are a few reasons for using linters:

- - Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
- - The entire codebase looks like written by one person.
- - Programmers can focus on solving problems, instead of cleaning up the code.

- Good job with using Microverse README template but let's remove the text pieces informing you about how you customize your README file. Those are below.
Screenshot 2023-03-22 at 10 46 12 PM
Screenshot 2023-03-22 at 10 46 19 PM
Screenshot 2023-03-22 at 10 46 38 PM
Screenshot 2023-03-22 at 10 46 53 PM
Screenshot 2023-03-22 at 10 47 00 PM
Screenshot 2023-03-22 at 10 47 30 PM
Screenshot 2023-03-22 at 10 47 37 PM

- Please remove the templated texts and add your custom content. For this project we use HTML&CSS.
Screenshot 2023-03-22 at 10 46 31 PM
- Since we don't have deployment for this project, please remove this section entirely. 👍 
Screenshot 2023-03-22 at 10 46 45 PM
- You can remove this section entirely since there will be no future parts of this project. 👍 
Screenshot 2023-03-22 at 10 47 18 PM





_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (didierganthier) [reply README.md]: Well done on adding the License file 💯 it makes the project look really more professional but the issue is that your link 🔗 is broken, the reason being that your file is named `LICENSE.md` and the link is looking for a file named `LICENSE` 📄 

Screenshot 2023-03-24 at 07 37 24

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @Braingenius2 ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by algerina — APPROVED** [review]
Hi @Braingenius2 ,

Great job so far! 

May I highlight the following: 
- [x] Your Readme looks fine👏  ✔️ 
- [x] The project is set correctly✔️ 
- [x] No Linter errors✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**